database/loadVolumeData.js

```diff
@@ -0,0 +1,243 @@
+'use strict'
+
+/* /bench/util/volumeRunner.js needs to have same values as here for teams, users etc. */
+const NUM_TEAMS = 500 // total number of teams
+
+const USER_START_ID = 1 // user start id, we may want a few super users
+const TEAM_START_ID = 7 // user start id, so as not to interfere with other test data
+const SUB_TEAM_MOD = 100 // 1 parent for every X-1 teams
+const NUM_USERS_PER_TEAM = 100 // put this many users in each team
+const NUM_POLICIES_PER_TEAM = 10 // :-|
+// const CREATE_INDEX = true
+
+const path = require('path')
+const pg = require('pg')
+const fs = require('fs')
+const config = require('../lib/config/build-all')()
+
+if (!config.get('local')) {
+  console.error('ERROR: You are trying to load test data in the database while not in local environment.')
+  process.exit(1)
+}
+
+const client = new pg.Client(config.get('pgdb'))
+
+var startTime
+var endTime
+
+function loadVolumeDataBegin (callback) {
+  startTime = Date.now()
+  console.log('loadVolume data started at ' + startTime)
+
+  // load original test data also..., should not interfere
+  console.log('loading existing fixtures from: ' + path.join(__dirname, '/testdata/fixtures.sql'))
+  let fixturesSQL = fs.readFileSync(path.join(__dirname, '/testdata/fixtures.sql'), 'utf8')
+
+  client.connect(() => { // connect first, then set off daisy chain of queries on success
+    client.query(fixturesSQL, (err) => {
+      if (err) {
+        callback(err)
+      } else {
+        endTime = Date.now()
+        logInColour('successfully loaded original fixtures')
+        loadTeams(callback, 'CONCH') // loads load everything into WONKA org
+      }
+    })
+  })
+}
+
+function loadTeams (callback, orgId) {
```

Comment:
the node convention is to have the callback param last, i.e. `(orgId, callback)`